### PR TITLE
cream.finance

### PR DESCRIPTION
**Pull request by shmuel-web**

### CODE REVIEW

File: backgroundJobs/ParseCream.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+class CreamParser extends Compound {
+    constructor() {
+        const compoundInfo = Addresses.creamAddress
+        const network = 'BSC'
+        const web3 = new Web3(process.env.BSC_NODE_URL)
+        super(compoundInfo, network, web3, 24 * 5)
+    }
+
+    async collectAllUsers() {
```
Comment (yaronvel) [inline backgroundJobs/ParseCream.js:16]: this is obsolete now, delete it.

File: backgroundJobs/priceFetcher.js [hunk 2]
```diff
@@ -55,6 +55,86 @@ const specialAssetPriceFetchers = {
     //CAN (cannon)
     return "0.0000000000000001"
   },
+  BSC_0xd4CB328A82bDf5f03eB737f37Fa6B370aef3e888: async (web3, network, stakedTokenAddress) => {
```
Comment (yaronvel) [inline backgroundJobs/priceFetcher.js:58]: why don't you call the general price function here?
the one that calls coingecko.

File: backgroundJobs/CompoundParser.js [hunk 3]
```diff
@@ -142,7 +142,7 @@ class Compound {
                 console.log("getting underlying")
                 const underlying = await ctoken.methods.underlying().call()
                 price = await getPrice(this.network, underlying, this.web3)
-                if(price.toString() == "0") {
+                if(price.toString() == "0" && this.network === "ETH") {
```
[the remark this answers is not in the document]
Comment (shmuel-web) [reply backgroundJobs/CompoundParser.js:145]: the process exits if there is no price
and then some bad debt calc will not update
and then we can debug the price fetcher to fix it

File: backgroundJobs/priceFetcher.js (hunk 2 above)
Comment (shmuel-web) [reply backgroundJobs/priceFetcher.js:58]: as discussed this is a different API

File: backgroundJobs/ParseCream.js (hunk 1 above)
Comment (shmuel-web) [reply backgroundJobs/ParseCream.js:16]: yes

**Pull request closed without merging**